app/2.0/docs/upgrade.md

```diff
@@ -89,35 +89,35 @@ Follow these steps to update your bower dependencies:
 
     | Component | Version |
     |-----------|---------|
-    | Polymer   | `^2.0.0-rc.3` |
-    | webcomponentsjs | `^1.0.0-rc.7` |
-    | web-component-tester | `^6.0.0-prerelease.6` |
-    | Polymer elements | `2.0-preview` |
+    | Polymer   | `^2.0.0` |
+    | webcomponentsjs | `^1.0.0` |
+    | web-component-tester | `^6.0.0` |
+    | Polymer elements | `2.0.0` |
 
-    Note that Polymer, webcomponentsjs, and web-component-tester have prerelease tags. You can use
-    a range like `^2.0.0-rc.3` to get all future prerelease tags. To use prerelease elements,
-    you need to refer to the `2.0-preview` branch. To get  updates to elements, you may need to remove
-    your `bower_components` folder and reinstall all components.
+    A list of 2.0-compatible Polymer elements can be found on the [About Polymer 2.0](/{{{polymer_version_dir}}}/docs/about_20#elements-2) page.
+    To get updates to elements, you may need to remove your `bower_components` folder
```

Comment:
After this change, the elements are tagged normally, so there's no difference from the other components. The only thing we should say here is that if they had one of the release candidates installed, they should remove the `bower_components` folder.